test/consume_test.py

```diff
@@ -46,6 +46,8 @@ def user_loop(request):
 )
 def user_mount(request):
     backend = validate_backend(request.param)
+    if backend.name == "mount-4k-ext2":
+        pytest.xfail("block size detection broken since kernel 5.5")
```

Comment:
Depending on backend name is too fragile. Can we skip the test when file system type is ext2?